pkg/services/ngalert/api/api_convert_prometheus.go

```diff
@@ -479,6 +482,7 @@ func (srv *ConvertPrometheusSrv) convertToGrafanaRuleGroup(
 			},
 			KeepOriginalRuleDefinition: util.Pointer(keepOriginalRuleDefinition),
 			EvaluationOffset:           &srv.cfg.PrometheusConversion.RuleQueryOffset,
+			NotificationReceiver:       c.Req.Header.Get(notificationReceiver),
```

Comment:
I think asking for the rest of notification settings would be too much, wouldn't it? :)

Reply:
Absolutely, I'm thinking about the best way to do that :)